test/KregielniaTests.cpp

```diff
@@ -1,11 +1,46 @@
 #include <gtest/gtest.h>
 #include "Kregielnia.hpp"
+#include "StringToVectorConverter.hpp"
+#include <string>
 
 struct KregielniaTests : public ::testing::Test
 {
 };
 
+int sumVector(std::vector<int> vec){
```

Comment:
not unified formatting. In other places there are opening braces { in next line

Reply:
Done